GitUI/UserControls/FileStatusList.cs

```diff
@@ -944,7 +944,7 @@ private void UpdateFileStatusListView(bool updateCausedByFilter = false)
                     group = new ListViewGroup(name)
                     {
                         // Collapse some groups for diffs with common BASE
-                        CollapsedState = i.Statuses.Count <= 7 || GitItemStatusesWithDescription.Count < 3 || i == GitItemStatusesWithDescription[0]
+                        CollapsedState = i == GitItemStatusesWithDescription[0] || (i == GitItemStatusesWithDescription[1] && !i.Summary.Contains("BASE"))
```

Comment:
This will collapse Staged in stash too
If BASE is configured in GetDiffStatus, it need to be used here too